resources/ahk/TradeMacro.ahk

```diff
@@ -1374,7 +1372,7 @@ TradeFunc_GetMeanMedianPrice(html, payload){
 		}
 		
 		; replace "
-		StringReplace, Currency, Currency, ", , All
+		StringReplace, Currency, Currency, ", , All			; "
```

Comment:
Oh that part was just breaking my editor because of the " on that line
so I just added an end quote in comment   ; "